.github/workflows/verify-dashboards-alerts.yaml

```diff
@@ -0,0 +1,135 @@
+name: Verify Dashboards and Alerts OK
+
+on: 
+  push:
+    branches: main
+    paths:
+       # Dashboards
+       - examples/dashboards/app_developer.json
+       - examples/dashboards/business_user.json
+       - examples/dashboards/platform_engineer.json
+       # Alerts
+       - examples/alerts/prometheusrules_policies_missing.yaml
+       - examples/alerts/slo-availability.yaml
+       - examples/alerts/slo-latency.yaml
+jobs:
+  verify-dashboards-alerts:
+    name: Verify Dashboards and Alerts OK
+    runs-on: ubuntu-latest
+    defaults: 
+      run:
+        shell: bash -eo pipefail {0}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run Quickstart
+        run: |
+          sudo chown $(whoami) .
```

Comment:
Why is this needed?

Reply:
It seemed that when I didn't use istio flag that the mgc folder being brought in had perm issues. With the new istio flag, this isn't needed at all. Nice catch !

Reply:
https://github.com/Kuadrant/kuadrant-operator/actions/runs/9837193514/job/27154462311